rpc_jobs/bump_snapshots.yml

```diff
@@ -0,0 +1,134 @@
+- job:
+    name: 'Bump-Snapshot-Images'
+    project-type: pipeline
+    concurrent: false
+    properties:
+      - build-discarder:
+          days-to-keep: 30
+    parameters:
+      - rpc_gating_params
+    triggers:
+      - timed: "@daily"
+    dsl: |
+      library "rpc-gating@${RPC_GATING_BRANCH}"
+      common.globalWraps(){
+        dir("releases") {
+          git branch: 'master', url: 'https://github.com/rcbops/releases'
+        }
+        // NOTE(mattt): We re-clone rpc-gating here because the existing clone
+        // has cruft in it which we don't want to have committed in our
+        // subsequent PR
+        dir("rpc-gating-master") {
+          git branch: 'master', url: 'https://github.com/rcbops/rpc-gating'
+
+          def componentData = readFile "${env.WORKSPACE}/releases/components/rpc-openstack.yml"
```

Comment:
I'd actually been suggesting you make use of `component` the CLI tool, not read the YAML directly, but I guess it works.